engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/EqualsAtomNode.java

```diff
@@ -81,7 +81,7 @@ boolean equalsAtoms(
         "cachedComparator != null",
       },
       limit = "10")
-  boolean equalsAtoms(
+  boolean equalsAtomsWithCustomComparator(
```

Comment:
Does this rename has any functional meaning (e.g. have I made a mistake when introducing the `EqualsAtomNode`?) or is it just stylistic? I hope the latter...

Reply:
This is just a stylistic issue. I thought that having `equalsAtomsWithCustomComparator`  and `equalsAtomWithDefaultComparator` specializations is better than two specializations with the same name `equalsAtoms` that have different guards.